applications/API/views.py

```
@@ -0,0 +1,676 @@
+#                 examples=[
+#                     OpenApiExample(
+#                         "Category Insults",
+#                         value=[
+#                             {
+#                                 "count": 116,
+#                                 "next": "http://127.0.0.1:8888/api/insults/?category=F&page=2",
+#                                 "previous": None,
+#                                 "results": [
+#                                     {
+#                                         "reference_id": "GIGGLE_OTc3",
+#                                         "content": "“Yo Momma’s so fat that when she walked past the TV, I missed three episodes.",
```

Ruff (0.12.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)